tests/ttnn/unit_tests/operations/test_full.py

```diff
@@ -0,0 +1,102 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+
+import torch
+import torch.nn as nn
+import ttnn
+from models.utility_functions import comp_allclose
+from loguru import logger
+
+from tests.ttnn.utils_for_testing import assert_equal
+
+
+@pytest.mark.parametrize(
+    "input_shape",
+    [
+        [32, 32],  # single tile
+        [5, 17, 31],  # multiple tiles
+    ],
+)
+@pytest.mark.parametrize(
+    "fill_value",
+    [3, -1],
+)
+def test_full_int(device, input_shape, fill_value):
+    torch_any = torch.randint(0, 100, (input_shape), dtype=torch.int32)
+    torch_output_tensor = torch.full(input_shape, fill_value)
+
+    any = ttnn.from_torch(torch_any, device=device, layout=ttnn.TILE_LAYOUT)
+    output_tensor = ttnn.moreh_full(input_shape, fill_value, any)
+    assert ttnn.is_tensor_storage_on_device(output_tensor)
+    output_tensor = ttnn.to_torch(output_tensor)
+
+    assert_equal(torch_output_tensor, output_tensor)
+
+
+@pytest.mark.parametrize(
+    "input_shape",
+    [
+        [32, 32],  # single tile
+        [5, 96, 64],  # multiple tiles
+        [3, 91, 67, 77],  # not multiple of 32
+    ],
+)
+@pytest.mark.parametrize(
+    "fill_value",
+    [0.15, -1.2],
+)
+@pytest.mark.parametrize(
+    "dtype",
+    [
+        torch.bfloat16,
```

Comment:
other types are not supported at the moment?

Reply:
For now, we only support float32, bfloat16, and int.